docs/architecture/cloud-native/definition.md

```diff
@@ -8,7 +8,7 @@ ms.date: 08/20/2019
 
 [!INCLUDE [book-preview](../../../includes/book-preview.md)]
 
-Stop what you’re doing and text 10 of your colleagues. Ask them to define the term “Cloud Native.” Good chance you’ll get eight different answers. Interestingly, six months from now, as cloud-native technologies and practices evolve, so will their definition.
+Stop what you're doing and text 10 of your colleagues. Ask them to define the term "Cloud Native". Good chance you'll get eight different answers. Interestingly, six months from now, as cloud-native technologies and practices evolve, so will their definition.
```

Comment:
Six months from when ?
I don't think this whole sentence has a good wording.

Reply:
@robvet Can you address this?

Reply:
A quick fix could be to say “six months later”

Reply:
I think, we should drop the entire sentence "Interestingly, six months..". The definition does not change, it's the perception that change and their understanding gets better. I was hoping Rob will come up with an interesting sentence to conclude the para.